konduit-serving-cli/src/main/java/ai/konduit/serving/cli/launcher/command/build/extension/model/Profile.java

```diff
@@ -0,0 +1,185 @@
+/*
+ *  ******************************************************************************
+ *  * Copyright (c) 2020 Konduit K.K.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Apache License, Version 2.0 which is available at
+ *  * https://www.apache.org/licenses/LICENSE-2.0.
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  * License for the specific language governing permissions and limitations
+ *  * under the License.
+ *  *
+ *  * SPDX-License-Identifier: Apache-2.0
+ *  *****************************************************************************
+ */
+
+package ai.konduit.serving.cli.launcher.command.build.extension.model;
+
+import ai.konduit.serving.vertx.config.ServerProtocol;
+import lombok.Data;
+import org.apache.commons.lang3.SystemUtils;
+import org.nd4j.shade.jackson.annotation.JsonGetter;
+import org.nd4j.shade.jackson.annotation.JsonSetter;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Data
```

Comment:
`@Data` generates getters/setters, which we probably don't want - i.e., no point having a `getComputeDevice()` and a `computeDevice()`.
So probably just `@EqualsAndHashCode` and `@ToString` needed here